data_dir/cs_lwt_perf_small.yaml

```diff
@@ -0,0 +1,86 @@
+# Keyspace Name
+keyspace: cs_lwt_perf_small
+
+# The CQL for creating a keyspace (optional if it already exists)
+keyspace_definition: |
+  CREATE KEYSPACE cs_lwt_perf_small WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+
+# Table name
+table: targettable
+
+# The CQL for creating a table you wish to stress (optional if it already exists)
+table_definition: |
+  CREATE TABLE targettable (
+        pr1 timeuuid,
+        aux1 int,
+        aux2 bigint,
+        aux3 double,
+        aux4 float,
+        aux5 tinyint,
+        PRIMARY KEY(pr1)
+  ) WITH compaction = { 'class':'LeveledCompactionStrategy' }
+    AND comment='A test table for cs_lwt_perf_small'
+
+### Column Distribution Specifications ###
+
+columnspec:
+  - name: pr1
+    size: uniform(1..1k)
+    population: uniform(1..10240M)
+  - name: aux1
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux2
+    size: uniform(1..1k)
+    population: uniform(1..10m)
+  - name: aux3
+    size: uniform(1..1k)
+    population: uniform(1..10m)
+  - name: aux4
+    size: uniform(1..1k)
+    population: uniform(1..10m)
+  - name: aux5
+    size: uniform(1..254)
+    population: uniform(1..254)
+
+### Batch Ratio Distribution Specifications ###
+
+# Insert is used only to propagate data into the dataset. This is only query that supports batches, it is actually works only thru batches
+# In order to call it add it to the ops statement: 'ops(insert=1,...)'
+# It overrides custom query 'insert' if it is defined
+# batch size maximum is hardcoded to 65535, it can be limiting row number in partition for big row insertation, by setting 'partitions' key to fixed(X)
+insert: # SchemaInsert.java, does batched prepared statements insert
+  partitions: fixed(4)
+  select:    fixed(4)/10240M
+  batchtype: UNLOGGED
+
+# After running "ops(insert=1) n=1000000" you will get ~1m records with ~76 avg row size across ~847k partitions, table size will be ~73m
```

Comment:
Again, we should use much bigger dataset... at least 100gb

Reply:
This particular file for tests that are designed to be done against dataset that could fit into memory. So no need to have huge dataset here.

Reply:
ok, but if you use i3.2xlarge the memory is 64GB.
We need something that is much bigger than 73MB.

50GB sounds good to me.